File: intro-prog/lista 6/ex07.py
"""
7. Escreva uma função que recebe uma lista vet e um número value. A função deve retornar uma
outra lista, contendo os múltiplos de value que estão em vet
"""

def getMultiples(ls, value):
    multiples = []
    for elem in ls:
        if elem % value == 0:
            multiples.append(elem)
    return multiples

def main():
    print(f"{getMultiples([2, 4, 6, 7, 3], 2)}")

main()

File: Algoritmos/Lista 2/ex07.c
/*7. Escreva uma função que recebe um vetor vet de tamanho n contendo números inteiros positivos e
negativos. A função deve inverter o sinal dos números negativos, passando-os para positivo*/

#include <stdio.h>

void setPositive(int size, int vet[]){
    for(int i = 0; i < size; i++){
        if(vet[i] < 0){
            vet[i] = -vet[i];
        }
        printf("%d\n", vet[i]);
    }
}

int main(){
    int vet[] = {-1, 2, -4, -6, 9};
    setPositive(5, vet);
}